AllASPFuncs.py

```diff
@@ -1174,17 +1189,20 @@ def build_lightcurve(ID, exposures, sn_path, confusion_metric, flux,
     band (str): the bandpass of the images used
 
     Returns:
-    lc: a pandas dataframe containing the lightcurve data
-    Notes:
-    1.) This will soon be ECSV format instead
-    2.) Soon I will turn many of these inputs into environment variable and they
-    should be deleted from function arguments and docstring.
+    lc: a QTable containing the lightcurve data
     '''
 
     detections = exposures[np.where(exposures['DETECTED'])]
     parq_file = find_parq(ID, path = sn_path, obj_type = object_type)
     df = open_parq(parq_file, path = sn_path, obj_type = object_type)
 
+    mags, magerr = calc_mags_and_err(flux, sigma_flux, band)
+    zeros = np.zeros_like(detections['realized flux'])
```

Comment:
I would write these lines as
```
sim_sigma_flux = 0
sim_realized_mag, _ = calc_mags_and_err(detections["realized flux"], sim_sigma_flux, band)
sim_true_mags, _ = calc_mags_and_err(detections["true flux"], sim_sigma_flux, band)
```

You can use a scalar `0` instead of constructing an array.